xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-webjar/src/main/webjar/searchSuggest.js

```diff
@@ -106,19 +106,22 @@ var XWiki = (function (XWiki) {
      * Submits the form or go to a selected page according to selection.
      */
     onSuggestionSelected: function(event) {
-      if (event.memo.suggest == this.suggest) {
+      if (event.memo.suggest === this.suggest) {
         event.stop();
-        // Also stop the browser event that triggered the custom "xwiki:suggest:selected" event.
-        if (event.memo.originalEvent) {
-          Event.stop(event.memo.originalEvent);
-        }
         if (!event.memo.url) {
+          // Stop the browser event that triggered the custom "xwiki:suggest:selected" event.
+          if (event.memo.originalEvent.type === 'click' && event.memo.originalEvent) {
```

Comment:
```suggestion
          if (event.memo.originalEvent && event.memo.originalEvent.type === 'click') {
```

I don't understand why you check that the `originalEvent` property exists **after** accessing a property of it. I also don't understand why this is only stopping `click` events. Shouldn't this stop all events in this case?